features/step_definitions/common_steps.rb

```diff
@@ -181,10 +181,9 @@
   breadcrumb.text().should == breadcrumb_text
 end
 
-Then /^I (don't |)see the '(.*)' link$/ do |negative, link_text|
-  page.should have_link(link_text) if negative.empty?
-
-  page.should_not have_link(link_text) unless negative.empty?
+Then /^I (don't |)see the '(.*)' (button|link)$/ do |negative, selector_text, selector_type|
+page.should have_selector(:link_or_button, selector_text) if negative.empty?
```

Comment:
Previously, if someone had converted a link to some other kind of element, then the test would presumably have failed.

Now, the test passes if the element is there are all, even if it isn't either a button or a link?

Seems like maybe we need a way to test based on selector_type (at which point, I'm not completely certain that doing this all in one function is the right thing, not sure :) ).